contrib/firewalld/template.py

```diff
@@ -0,0 +1,69 @@
+pkgname = "firewalld"
+pkgver = "2.1.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--disable-sysconfig", "--disable-systemd"]
+configure_gen = ["./autogen.sh"]
+make_cmd = "gmake"
+hostmakedepends = [
+    "automake",
+    "gmake",
+    "gtk-doc-tools",
+    "intltool",
+    "libxml2-progs",
+    "pkgconf",
+    "python",
+]
+makedepends = [
+    "gettext-devel",
+    "glib-devel",
+]
+depends = [
+    "python-dbus",
+    "python-gobject",
+    "python-nftables",
+]
+pkgdesc = "Stateful zone based firewall daemon with D-Bus interface"
+maintainer = "miko <mikoxyzzz@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://firewalld.org"
+source = f"https://github.com/{pkgname}/{pkgname}/releases/download/v{pkgver}/{pkgname}-{pkgver}.tar.bz2"
+sha256 = "22e3c700c2e4867796e9c22cf929cba4b2c143c8884240cfb9f3ef731366ff71"
+# tests don't work in our build env
+options = ["!check"]
+
+
+def do_prepare(self):
```

Comment:
do you need this at all? also use named arguments

Reply:
does the build system just glob everything from that place?

Reply:
build system doesn't even seem to want to install the icons for whatever reason. i have to manually install them, in which case it's just easier to nuke firewall-applet's icons so that i can just run install_files on the icon dirs

Reply:
gotcha